official/utils/misc/model_helpers.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2018 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Miscellaneous functions that can be called by models."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import numbers
+
+import tensorflow as tf
+
+
+def past_stop_threshold(stop_threshold, eval_metric):
```

Comment:
As discussed offline, probably should have a similar function which stop when a metric is smaller that the threshold.

Reply:
It's pretty simple to add a flag in theory, but in practice, I don't know that we have a use-case yet for a lower-is-better metric. @petermattson , is any target metric decreasing?

Reply:
For performance purposes, crossing the line is acceptable, since all the benchmark implementations play by the same rules. The only question is timing instability, and we're addressing that through multiple runs. We can revisit if that's insufficient.